authentication/epic.js

```diff
@@ -15,112 +16,125 @@ const {
 module.exports = combineEpics(initEpic, signInEpic, logOutEpic, registerEpic)
 
 exports.initEpic = initEpic
-function initEpic () {
-  return Rx.Observable.of(signIn(createCid()))
-    .delay(0) // apparently needs delay otherwise action lost
+function initEpic (action$) {
+  return of(signIn(createCid()))
+  // .pipe(
+  //   delay(0) // apparently needs delay otherwise action lost
```

Comment:
are we _sure_ about removing this?

Reply:
no i am not sure 😅 everything seems to be working without it, i see the `SIGN_IN` actions i expect to see, but perhaps you have more context there @ahdinosaur ?